framer/Events.coffee

```diff
@@ -25,15 +25,14 @@ Events.enableEmulatedTouchEvents = (enable=true) ->
 		Events.TouchStart = "touchstart"
 		Events.TouchEnd = "touchend"
 		Events.TouchMove = "touchmove"
+	Events.Click = Events.TouchEnd
 
 Events.enableEmulatedTouchEvents(false)
 
-# Let's make sure the touch events work on desktop too
-if not Utils.isTouch()
+# Let's make sure the touch events work with a mouse too
+if Utils.isMouse()
```

Comment:
This seems like it still should be `not Utils.isTouch()`? (only enable the emulator if there’s no touch)